en/advanced_config/advanced_mc_position_tuning.md

```diff
@@ -1,96 +1,103 @@
-<!-- Fixes page load bug in notation, courtesy https://github.com/GitbookIO/plugin-mathjax/issues/34#issuecomment-349453673 -->
-<script>
-gitbook.events.bind("page.change", function() {
-    MathJax.Hub.Queue(["Typeset",MathJax.Hub]);
-}
-</script>
-
 # Advanced Multicopter Position Control Tuning
- The objective of this document is to give an overview of the higher-level position-control tuning parameters for multicopters. Higher-level tuning applies to any parameter that has an effect on
- the desired position/velocity setpoint as opposed to [lower-level tuning](pid_tuning_guide_multicopter.md) that influences the tracking.
- The set of higher-level tuning parameters can be split into two sections: tunig parameters for manual control position mode and tuning parameters for auto. Some parameters will have an effect on both
- modes. 
 
-> **Warning** This guide is for advanced users/experts. 
+This document provides an overview of the higher-level position-control tuning parameters for multicopters.
+
+> **Warning** This guide is for advanced users/experts.
+
+<span></span>
+> **Tip** Follow the instructions in the [Multicopter PID Tuning Guide](https://docs.px4.io/en/advanced_config/pid_tuning_guide_multicopter.html) *before* doing any of the higher-level related control tuning described here. Do not use the advanced position control tuning parameters to fix bad tracking or vibration!
+
+## Overview
 
-## Short Background
-The position controller consists of an outer P-position control loop and an inner PID-velocity control loop. Depending on the control mode either
-the velocity control loop only is active or both. For the remaining part of this tutorial the term **position-control** represents the position and velocity control loop
-activated and **velocity-control** when just the velocity control loop is in use.
+Higher-level tuning applies to any parameter that has an effect on the desired position/velocity setpoint, in contrast to [lower-level tuning](pid_tuning_guide_multicopter.md) that influences the tracking.
```

Comment:
@Stifael I don't understand the difference. Lower level tuning influences the tracking of "what"? I "think" you might be saying that lower level tuning influences how well the autopilot tracks setpoints, but I'm not then sure how the higher level tuning differs.

Reply:
> lower level tuning influences how well the autopilot tracks setpoints

That is correct. The goal of the P/PID controller is to keep the error between the desired setpoint and estimate of the vehicle state as small as possible. Wrong P/PID Gains can lead to instability. 
The higher level tuning, on the other hand, has no influence on the tracking at all. It just adjusts the setpoint. For instance, instead of step input, you will have a ramp input. Furthermore, it is not possible to make the system instable by adjusting the higher level tuning parameter. 
By the way, the name "higher level" is something that I made up. The point is that there has to be a clear difference between the two kind of parameters: One is for keeping the error small, the other set only effects the setpoint